### PR TITLE
Fix Adding "ToDo" note crash when no active object

### PR DESCRIPTION
**Pull request by SNoiraud**
Fixes [#10795](https://gramps-project.org/bugs/view.php?id=10795)

### CODE REVIEW

File: gramps/plugins/gramplet/todo.py [hunk 1]
```diff
@@ -184,13 +185,20 @@ def new_clicked(self, obj):
...
+            from gramps.gui.editors import EditNote
+            note = Note()
+            note.set_type(NoteType.TODO)
+            try:
+                EditNote(self.gui.dbstate, self.gui.uistate, [], note, self.created)
+            except AttributeError:
+                pass
+        else:
+            WarningDialog(_("No active object"),
+                          _("No active object") + _(":") + _(nav_type),
```
Comment (Nick-Hall) [inline gramps/plugins/gramplet/todo.py]: The string "No active object" appears in the title bar, primary message and secondary message.  Can we think of something more helpful?  For example:  "First select the object to which you want to attach a note".

Comment (SNoiraud) [reply gramps/plugins/gramplet/todo.py]: I changed the message as you said.

Comment (sam-m888) [reply gramps/plugins/gramplet/todo.py]: New Example message

example message

Comment (sam-m888) [reply gramps/plugins/gramplet/todo.py]: @Nick-Hall Is this message ok to you?

**Comment by sam-m888** [comment]
@SNoiraud Please rebase.

**Pull request merged**